rcgen/src/certificate.rs

```diff
@@ -607,6 +616,21 @@ impl CertificateParams {
 		})
 	}
 
+	/// Generate and serialize a certificate signing request (CSR).
+	///
+	/// The constructed CSR will contain attributes based on the certificate parameters,
+	/// and include the subject public key information from `subject_key`. Additionally,
+	/// the CSR will be self-signed using the subject key.
+	///
+	/// Note that subsequent invocations of `serialize_request()` will not produce the exact
+	/// same output.
+	pub fn serialize_request(
```

Comment:
Nit: `serialize_request()` should appear before `serialize_request_with_attributes()`.

Reply:
(Done)